posix.mak

```diff
@@ -375,6 +383,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+# Target for quickly debugging a single module
+# For example: DEBUGGER=ddd BUILD=debug make -f posix.mak std/format.debug
```

Comment:
Should be `make -f posix.mak DEBUGGER=ddd std/format.debug` except environmental variables are used.